opacclient/libopac/src/main/java/de/geeksfactory/opacclient/objects/AccountItem.java

```diff
@@ -0,0 +1,148 @@
+package de.geeksfactory.opacclient.objects;
+
+public abstract class AccountItem {
+    private long account;
+    private String title;
+    private String author;
+    private String format;
```

Comment:
I don't like the thought of maintaining two options, but the benefits seem to be worth it.